mesh/mesh.cpp

```diff
@@ -10672,14 +10672,15 @@ void Mesh::PrintXG(std::ostream &os) const
    os << flush;
 }
 
-void Mesh::Printer(std::ostream &os, std::string section_delimiter) const
+void Mesh::Printer(std::ostream &os, std::string section_delimiter,
+                   std::string const& comments) const
```

Comment:
In most places in MFEM we put the `const` before the type -- it's probably better to stick to that convention here and in other changes in this PR.

Reply:
I will change it if we decide to keep this interface.